delta/plugins/archive/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/ArchiveDownloadSpec.scala

```diff
@@ -55,6 +55,7 @@ class ArchiveDownloadSpec
     with AnyWordSpecLike
     with Inspectors
     with EitherValuable
+    with CatsRunContext
```

Comment:
It would be better if we would not need two traits `CatsRunContext` and `CatsIOValues` here

Reply:
I don't disagree as such, but I think there are times where you need the context but not the values. I think it would be best to address this by looking at some of our base test classes and putting this in there so it's more hidden. Do you think we can do that just so we can put off dealing with it now?

Reply:
It can be done in further PR, can you create an issue for it ?

Reply:
https://github.com/BlueBrain/nexus/issues/4371